coralibre-sdk/sdk/src/main/java/org/coralibre/android/sdk/internal/misc/Handshake.java

```diff
@@ -7,7 +7,7 @@
  *
  * SPDX-License-Identifier: MPL-2.0
  */
-package org.coralibre.android.sdk.internal.database.models;
+package org.coralibre.android.sdk.internal.misc;
 
 import org.coralibre.android.sdk.internal.crypto.EphId;
```

Comment:
This file can be removed compleatly. (If it's not causing to much trouble removing it. Otherwise we should remove it later in al deep clean PR).

Reply:
Let's create a new issue for cleaning these parts after the PR is done. Cleaning of these classes has not much to do with the database stuff.